src/constants/color.ts

```diff
@@ -0,0 +1,18 @@
+export const COLORS = {
+  WHITE_50: "#FFFFFF",
+  WHITE_100: "#ecebf1",
+  WHITE_125: "#e5e5e5",
+  WHITE_150: "#d3d3d3",
+  WHITE_200: "#D2D2D2",
+  GRAY_100: "#bababa",
+  GRAY_200: "#9ca3af",
+  GRAY_250: "#737373",
+  GRAY_275: "#575757",
+  GRAY_300: "#525252",
+  BLACK_100: "#000000",
+  YELLOW_100: "#cbba64",
+  GREEN_100: "#04c09e",
+  GREEN_200: "#94daca",
+  RED_100: "#ff0000",
+  RED_200: "#ff6d6d",
+};
```

Comment:
상수는~~ readonly로 만듭시다~~~ 
```suggestion
export const COLORS = {
  WHITE_50: "#FFFFFF",
  WHITE_100: "#ecebf1",
  WHITE_125: "#e5e5e5",
  WHITE_150: "#d3d3d3",
  WHITE_200: "#D2D2D2",
  GRAY_100: "#bababa",
  GRAY_200: "#9ca3af",
  GRAY_250: "#737373",
  GRAY_275: "#575757",
  GRAY_300: "#525252",
  BLACK_100: "#000000",
  YELLOW_100: "#cbba64",
  GREEN_100: "#04c09e",
  GREEN_200: "#94daca",
  RED_100: "#ff0000",
  RED_200: "#ff6d6d",
} as const;
```

Reply:
아차차....22 read only 좋아요!!

- [refactor: 상수에 readonly 추가](https://github.com/usageness/react-payments/pull/1/commits/0d59a0ce840662ffa96110d8eeb0245a3a445fe3)